cfme/tests/middleware/jdr_report_methods.py

```diff
@@ -0,0 +1,30 @@
+from cfme.utils.wait import wait_for
+from cfme.middleware.jdr_report import JDRReport
```

Comment:
**REQUIRED**
Please follow style guides for imports http://cfme-tests.readthedocs.io/guides/dev_guide.html